### PR TITLE
Adding migration guide for breaking changes in the AzureStack module …

### PR DESCRIPTION
**Pull request by bganapa**
…version 1.7.0

## Checklist

- [x] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [ ] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md#updating-the-change-log)
- [x] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (Microsoft internal partners only)
    - [ ] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/help-generation.md)

### CODE REVIEW

File: documentation/migration-guides/Stack/azurestack.1.7.0.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
+
+This document serves as both a breaking change notification and migration guide for consumers of the AzureStack powershell version 1.7.0. This release supports Azurestack 1901 update or later
+
+## Table of Contents
+- [Breaking changes to Backup Admin cmdlets](#breaking-changes-to-backup-admin-cmdlets)
+- [Breaking changes to Fabric Admin cmdlets](#breaking-changes-to-fabric-admin-cmdlets)
+
+### AzureRm.Profile dependency
```
[the remark this answers is not in the document]
Comment (bganapa) [reply documentation/migration-guides/Stack/azurestack.1.7.0.md]: Ah, It is not supposed to be there.. I am removing it

**Comment by daozha** [comment]
Backup part looks good to me. Thanks.

**Comment by vladimir-shcherbakov** [comment]
signing: https://azuresdkci.westus2.cloudapp.azure.com/view/PowerShell/job/ps-sign-esrp/

**Comment by bganapa** [comment]
@vladimir-shcherbakov the changes are in documentation folder, it should not need ps-sign run

**Pull request merged**